docs/tutorials/sharing-environments.md

```diff
@@ -124,14 +124,38 @@ $ flox push
 ```
 
 !!! note "Note"
-      Right now, only environment owners can push edits to their environments.
+    Right now, only environment owners can push edits to their environments.
 
 
 [flox_push]: ../reference/command-reference/flox-push.md
 [flox_pull]: ../reference/command-reference/flox-pull.md
 [flox_activate]: ../reference/command-reference/flox-activate.md
 [floxhub_concept]: ../concepts/floxhub.md
 
+
+### Pulling a remote environment without connecting to FloxHub
+
+Sometimes you may want to create a FloxHub remote environment that serves as a template starting point for your project but will grow to have different environment definitions across multiple projects. 
+In this cases, you may want to [`flox pull --copy`][flox_pull] instead of [`flox pull`][flox_pull]. 
+
+[`flox pull --copy`][flox_pull], like [`flox pull`][flox_pull], will create a `.flox` folder to the directory you are in. However, this environment will **not be linked to FloxHub**.
```

Comment:
```suggestion
[`flox pull --copy`][flox_pull], like [`flox pull`][flox_pull], will create a `.flox` folder to the directory you are in. However, this environment will **not be linked to FloxHub**, instead it will be entirely local as if created from scratch using [`flox init`][flox_init] and can be pushed under a new name or owner.
```

or something similar, i think "not be linked to floxhub" is not making the implications obvious